patches/node/enable_crashpad_linux_node_processes.patch

```diff
@@ -27,13 +27,14 @@ index 73c1dc769542865bdf7a2a03c16cef141d3f4b05..35a096dcff6f7072391bccd7952da7e8
    args = [...execArgv, modulePath, ...args];
  
    if (typeof options.stdio === 'string') {
-@@ -613,6 +613,21 @@ function normalizeSpawnArguments(file, args, options) {
+@@ -613,6 +613,22 @@ function normalizeSpawnArguments(file, args, options) {
                      'options.windowsVerbatimArguments');
    }
  
 +  if (process.platform === 'linux') {
 +    if (ObjectPrototypeHasOwnProperty(options.env || {}, 'ELECTRON_RUN_AS_NODE') &&
```

Comment:
since options.env isn't necessarily set here, how does this work...? should this be `options.env || process.env`?

Reply:
Good question, I think so? Made the change in [b449baf](https://github.com/electron/electron/pull/36460/commits/b449baf781d9da8e7c0cdf4a38a27fd184670543) to use `process.env` instead of `{}`

Reply:
Ah, I actually see what happens earlier in the file - we set `options` to `kEmptyObject` if it's initially undefined.